src/Qbead.h

```diff
@@ -74,14 +74,17 @@ float sign(float x) {
   else return -1;
 }
 
+int roundToInt(float x) {
+  if (x > 0) return (int)(x + 0.5);
+  else return (int)(x - 0.5);
+}
+
 // z = cos(t)
 // x = cos(p)sin(t)
 // y = sin(p)sin(t)
-float phi(float x, float y, float z) {
-  float ll = x * x + y * y + z * z;
-  float l = sqrt(ll);
-  float phi = atan2(y, x);
-  return phi;
+// Return the angle in radians between the x-axis and the line to the point (x, y)
+float phi(float x, float y) {
```

Comment:
Do you mind adding a three-argument version of this that just dispatches to the two-argument version, in case this is already used in external code? No need to worry about overhead here, compilers are smart enough to remove unnecessary function calls (e.g. even in the old code the `sqrt` does not exist in the actual machine code as you can see here https://godbolt.org/z/EjxYsdjeK )

Reply:
I did check and update all usages in this repo. The overloaded function is added though.